### PR TITLE
Add feature usage observer

### PR DESCRIPTION
**Pull request by Soare-Robert-Daniel**
## Summary 

- Enabled the SDK telemetry for this product.
- The plugin settings are tracked via the Form submit event.
- Added block options tracking.

Also added the toggle for Telemetry since WP does not want telemetry by default.

image

## Testing

- Enable Telemetry in the settings.
- Check if something is broken.
- Post the test instance URL so we can check in the metabase for the saved events.

### CODE REVIEW

File: includes/admin/feedzy-rss-feeds-admin.php [hunk 1]
```diff
@@ -582,6 +594,8 @@ private function save_settings() {
 		}
 		$post_tab = isset( $_POST['tab'] ) ? filter_input( INPUT_POST, 'tab', FILTER_UNSAFE_RAW ) : '';
 
+		error_log( print_r( $_POST, true ) );
```
Comment (GrigoreMihai) [inline includes/admin/feedzy-rss-feeds-admin.php]: This seems like a leftover from testing, we should remove it.

**Review by GrigoreMihai — APPROVED** [review]
Apart from the comment mentioned everything looks good.

**Comment by vytisbulkevicius** [comment]
@Soare-Robert-Daniel, do you think we can integrate the question about agreeing to enable Telementary into the onboarding steps of Feedzy also? (and make it enabled if they agree)

I think it's quite a low chance for users to go in there in Settings and enabling on purpose it without us pointing to it in any way/asking them.

**Comment by Soare-Robert-Daniel** [comment]
> @Soare-Robert-Daniel, do you think we can integrate the question about agreeing to enable Telemetry into the onboarding steps of Feedzy also? (and make it enabled if they agree)

@vytisbulkevicius, We can do something like this for Import Post Onboarding.

image

**Comment by vytisbulkevicius** [comment]
Perfect, thanks @Soare-Robert-Daniel

**Comment by vytisbulkevicius** [comment]
@preda-bogdan, please give a look at this one if you have any suggestions, looking to include it in the release today. Thanks! 🙏

**Review by preda-bogdan — APPROVED** [review]

**Pull request merged**